Extension/cpptools.json

```diff
@@ -3,5 +3,10 @@
     "defaultIntelliSenseEngine": 100,
     "recursiveIncludes": 100,
     "gotoDefIntelliSense": 100,
-    "enhancedColorization": 100
+    "enhancedColorization": 100,
+    "supportExtensionVscodeVersion": {
+        "0.28.0": "1.44.0",
```

Comment:
I don't think we should add each extension version in here.  We only need to add an entry when the vscode dependency changes.  We could end up forgetting to update this when there is no dependency change.